### PR TITLE
setup for github pages

### PR DESCRIPTION
**Pull request by mcontre-g**
Changed title of website. Commented out the previous img line in index.html so that there is only one image in the webpage.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,7 +40,7 @@ <h1 id="title">
                     for the website to work on github pages and the other one
                     is for the node server -->
                     <img src="public/images/article_image.png" alt="Glenn Harvey" id="article_image">
-                    <img src="images/article_image.png" alt="Glenn Harvey" id="article_image">
+                    <!-- <img src="images/article_image.png" alt="Glenn Harvey" id="article_image"> -->
```
Comment (allenturner-os) [inline index.html:43]: Why not just remove this instead of commenting it out?

File: index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
     <body>
         <!-- TODO: add some add on top of page -->
         <div class="main_region sticky" id="top_bar">
-            <p>The Google Times</p>
+            <p>The Random News Website</p>
```
Comment (allenturner-os) [inline index.html:13]: "Generic News Website" might be a better name for this. What do you think?

Comment (joulesm) [reply index.html]: SGTM.

File: index.html [hunk 3]
```diff
@@ -8,12 +8,9 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
     </head>
     <body>
-        <!-- TODO: add some add on top of page -->
         <div class="main_region sticky" id="top_bar">
-            <p>The Google Times</p>
+            <p>Generic News Website</p>
         </div>
-        <!-- This script will serve as an add -->
-        <script id="banner_add"></script>
```
Comment (joulesm) [inline index.html:16]: nit: s/add/ad

File: index.html (hunk 1 above)
Comment (mcontre-g) [reply index.html]: Done

**Pull request merged**